### ISSUE TITLE
Translation issues in PDP

### ISSUE THREAD

**Issue by zenweasel** [reactioncommerce/reaction#2676]
### Expected behavior
![basic_reaction_product](https://user-images.githubusercontent.com/72819/29302598-118f0870-81b8-11e7-9212-0076126fa82f.png)

### Actual Behavior
![basic_reaction_product](https://user-images.githubusercontent.com/72819/29302610-329be7c2-81b8-11e7-884d-8c9e63388667.png)

### Steps to Reproduce the Behavior
1. Login as admin on either `development` or `marketplace` branch
1. Switch your language to German
1. Click on the option edit icon to open the side panel
1. Observe that no fields here are translated.

### Versions
```
Node: 7.10.0
NPM: 5.3.0
Meteor Node: 4.8.4
Meteor NPM: 4.6.1
Reaction CLI: 0.11.0
Reaction: 1.4.1
Reaction branch: marketplace
Docker: 17.06.0-ce
```

**Comment by awadesmoline** [comment]
I could not reproduce this bug on `development` but could reproduce it on `marketplace`.  I have raised a PR to resolve it #2680

**Comment by zenweasel** [comment]
Closed via #2680
